--- a/branches/trunk/Kernel/spot-on-kernel.qt5.pro
+++ b/branches/trunk/Kernel/spot-on-kernel.qt5.pro
@@ -35,7 +35,6 @@
 			  -Werror -Wextra -Wl,-z,relro \
 			  -Woverloaded-virtual -Wpointer-arith \
                           -Wstack-protector -Wstrict-overflow=4
-QMAKE_LFLAGS_RELEASE += -Wl,-rpath,/usr/local/spot-on/Lib
 QMAKE_EXTRA_TARGETS = libntru libspoton purge
 QMAKE_LFLAGS_RPATH =
 INCLUDEPATH	+= . ../. ../../../.
